tests/storage/psql_dos/test_backend.py

```diff
@@ -83,16 +57,19 @@ def test_get_unreferenced_keyset():
     repository_backend = aiida_backend.get_repository()
     repository_backend.delete_objects(keys)
 
-    with pytest.raises(
-        RuntimeError, match='There are objects referenced in the database that are not present in the repository'
-    ) as exc:
-        get_unreferenced_keyset()
+    expected_error = 'There are objects referenced in the database that are not present in the repository'
+    with pytest.raises(RuntimeError, match=expected_error) as exc:
+        storage_backend.get_unreferenced_keyset()
     assert 'aborting' in str(exc.value).lower()
 
 
 #yapf: disable
-@pytest.mark.parametrize(('kwargs', 'logged_texts'), (
+@pytest.mark.parametrize(
```

Comment:
Think this is autoformatted? Maybe then remove the `# yapf: disable`?

Reply:
Not sure what you mean. I choose this formatting manually for clarity, if I let `yapf` format it I get this, which I think is a bit more obtuse:

```python
@pytest.mark.parametrize(('kwargs', 'logged_texts'), (
    ({}, [' > live: True', ' > dry_run: False']),
    ({
        'full': True,
        'dry_run': True
    }, [' > live: False', ' > dry_run: True']),
    ({
        'extra_kwarg': 'molly'
    }, [' > live: True', ' > dry_run: False', ' > extra_kwarg: molly']),
))
```